backend/src-common/src/test/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandlerTest.java

```diff
@@ -67,7 +63,7 @@ public void setUp() throws Exception {
 
         List<Project> projects = new ArrayList<>();
 
-        Project p1 = new Project().setId("P1").setName("Project1").setBusinessUnit("AB CD EF").setCreatedBy("user1")
+        Project p1 = new Project().setId("P1").setName("Project1").setBusinessUnit("AB CD EF").setCreatedBy("user1").setAttachments(Collections.emptySet())
```

Comment:
this is cheating ;) I see no reason why it shouldn't work when project's attachments are null. Or is there one?
Also, you can't be sure that there aren't projects in the DB with null attachments.
I suggest instead to add an extra test to make sure that null attachments won't cause problems and won't delete accepted attachments, either.

This applies to line 76 and changes in ProjectHandlerTest, as well.